actionpack/lib/action_dispatch/system_test_case.rb

```diff
@@ -134,7 +134,10 @@ def self.start_application # :nodoc:
     #   driven_by :selenium, using: :firefox
     #
     #   driven_by :selenium, using: :headless_firefox
-    def self.driven_by(driver, using: :chrome, screen_size: [1400, 1400], options: {})
+    def self.driven_by(driver, using: nil, screen_size: [1400, 1400], options: {})
+      Capybara.session_name = options.fetch(:session_name) do
+        [driver, using].compact.join("_").to_sym
+      end
```

Comment:
This shouldn't be done in the `driven_by` method and rather should be done in the [driver class](https://github.com/rails/rails/blob/master/actionpack/lib/action_dispatch/system_testing/driver.rb) where all the other direct Capybara calls are.